setup.py

```diff
@@ -5,8 +5,8 @@
 
 
 install_requires = (
-    'djangorestframework>=2.4.3,<3',
-    'incuna_mail>=2.0.0,<3',
+    'djangorestframework>=2.4.4,<3',
+    'incuna_mail>=2.0.0,<=3.0.0',
```

Comment:
Well v3.1 should be backwards compatible, so if it works with v3, it should work with `<4`

Cool that it works with v2 as well